configs/rule_bindings/team-4-rule-bindings.yml

```diff
@@ -0,0 +1,71 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+rule_bindings:
+  T4_URI_DP_EMAIL_DUPLICATE:
+    entity_uri: dataplex://projects/dataplex-clouddq/locations/us-central1/lakes/amandeep-dev-lake/zones/raw/entities/contact_details
+    column_id: VALUE
+    row_filter_id: DATA_TYPE_EMAIL
+    rule_ids:
+      - NO_DUPLICATES_IN_COLUMN_GROUPS:
+          column_names: "contact_type,value"
+      - NOT_NULL_SIMPLE
+      - IN_REF_SET
+    metadata:
+      brand: one
+
+
+  T5_URI_BQ_EMAIL_DUPLICATE:
+    entity_uri: bigquery://locations/us-central1/lakes/amandeep-dev-lake/zones/raw/projects/dataplex-clouddq/datasets/clouddq_us_central1/tables/contact_details
+    column_id: VALUE
+    row_filter_id: DATA_TYPE_EMAIL
+    rule_ids:
+      - NO_DUPLICATES_IN_COLUMN_GROUPS:
+          column_names: "contact_type,value"
+      - NOT_NULL_SIMPLE
+      - IN_REF_SET
+    metadata:
+      brand: one
+
+  T6_URI_BQ_ELIDE_EMAIL_DUPLICATE:
+    entity_uri: bigquery://projects/dataplex-clouddq/datasets/clouddq_us_central1/tables/contact_details
```

Comment:
test that this only works if "zones" is provided in metadata_registry_defaults dataplex defaults

Reply:
todo: amandeep: add test_templates to test intermediate json and output sql